<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>controlShapesWidget</class>
 <widget class="QWidget" name="controlShapesWidget">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>1191</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>548</width>
        <height>1198</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="0" column="0">
        <widget class="QGroupBox" name="saveGroup">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QPushButton" name="saveControlsButton">
            <property name="toolTip">
             <string>Save current rigs control configuration into build config</string>
            </property>
            <property name="text">
             <string>Save Rig Control Shapes</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="mirrorControlsButton">
            <property name="toolTip">
             <string>Mirror Selected Control shapes. Only works for controls Prefixed with &quot;L&quot; or &quot;R&quot;</string>
            </property>
            <property name="text">
             <string>Mirror Rig Controls Shapes</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="builderExpanderTemp">
         <property name="toolTip">
          <string>Tools for modify existing control shapes</string>
         </property>
         <property name="text">
          <string>Control Shape Modifier</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="builderGroup">
         <property name="title">
          <string>Builder</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="3" column="0">
           <widget class="QPushButton" name="replaceShapesButton">
            <property name="toolTip">
             <string>Replaces last shape in selection with the first shape in the selection.</string>
            </property>
            <property name="text">
             <string>Replace Control Shape</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QPushButton" name="replaceByNameButton">
            <property name="toolTip">
             <string>Scans scene for new shapes with &quot;_LiftedShape&quot; prefix. And replaces their non-lifted counter parts with the new shapes.</string>
            </property>
            <property name="text">
             <string>Replace Control Shape by Name</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="addToShapeButton">
            <property name="toolTip">
             <string>Add selected shape to last shape in selection. Similar to combine shapes but works on non lifted shapes.</string>
            </property>
            <property name="text">
             <string>Add to Shape</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="combineControlsButton">
            <property name="toolTip">
             <string>Combine Selected shapes into one shape. Rig controls must be lifted before use</string>
            </property>
            <property name="text">
             <string>Combine Shapes</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="liftControlsButton">
            <property name="toolTip">
             <string>Lift selected control shapes so they can be modified safely.</string>
            </property>
            <property name="text">
             <string>Lift Control Shape</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="createControlsExpanderTemp">
         <property name="toolTip">
          <string>Control shape builder. Stored control shapes for easy use.</string>
         </property>
         <property name="text">
          <string>Create New Control Shapes </string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QGroupBox" name="serialiseControlShapeGroup">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QScrollArea" name="createControlsScroll">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
              <horstretch>100</horstretch>
              <verstretch>200</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>500</width>
              <height>500</height>
             </size>
            </property>
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOn</enum>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>475</width>
               <height>496</height>
              </rect>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>200</height>
              </size>
             </property>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="1" column="0">
               <widget class="QGroupBox" name="scrollGroup">
                <property name="title">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QGroupBox" name="groupBox">
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="title">
             <string/>
            </property>
            <layout class="QGridLayout" name="gridLayout_8">
             <item row="0" column="0">
              <widget class="QGroupBox" name="saveShapeGroup">
               <property name="autoFillBackground">
                <bool>true</bool>
               </property>
               <property name="title">
                <string>Save new control shape</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_6">
                <item row="0" column="1">
                 <widget class="QLineEdit" name="controlNameLineEdit">
                  <property name="toolTip">
                   <string>Name to store saved control shape under. If left blank the objects name will be used instead</string>
                  </property>
                  <property name="whatsThis">
                   <string/>
                  </property>
                  <property name="accessibleName">
                   <string/>
                  </property>
                  <property name="accessibleDescription">
                   <string/>
                  </property>
                  <property name="inputMethodHints">
                   <set>Qt::ImhNone</set>
                  </property>
                  <property name="inputMask">
                   <string/>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QPushButton" name="serialiseControlShapeButton">
                  <property name="toolTip">
                   <string>Save selected shape as new control shape or overide existing shape</string>
                  </property>
                  <property name="text">
                   <string>Serialise Control Shape</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="nameInputLabel">
                  <property name="text">
                   <string>Shape Name:</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QPushButton" name="tempRiggedControlShapeButton">
               <property name="toolTip">
                <string>Rigged control shapes for creation of custom control shapes</string>
               </property>
               <property name="autoFillBackground">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>Rigged Control Shapes</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QGroupBox" name="riggedControlShapesGroup">
               <property name="autoFillBackground">
                <bool>true</bool>
               </property>
               <property name="title">
                <string/>
               </property>
               <layout class="QGridLayout" name="gridLayout_7">
                <item row="0" column="0">
                 <widget class="QPushButton" name="riggedTwoHeadedArrowButton">
                  <property name="toolTip">
                   <string>Builds a rigged two headed arrow shape which can be used to create custom arrow shapes. </string>
                  </property>
                  <property name="text">
                   <string>Rigged Two-Headed Arrow</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="colorControlsExpanderTemp">
         <property name="toolTip">
          <string>Tools to color control shapes.</string>
         </property>
         <property name="text">
          <string>Color Control Shapes</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QGroupBox" name="colorControlsGroup">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="colorChangeLayout">
          <item row="1" column="0" colspan="2">
           <widget class="QPushButton" name="setColorButton">
            <property name="toolTip">
             <string>Set control color based on color selected above</string>
            </property>
            <property name="statusTip">
             <string>Set color of selected control shape</string>
            </property>
            <property name="text">
             <string>Set Color on Control</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QPushButton" name="colorPickerTemp">
            <property name="text">
             <string>colorPickerTemp</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="removeOutlinerColorButton">
            <property name="toolTip">
             <string>Removes outline color from selected object</string>
            </property>
            <property name="statusTip">
             <string>Remove color in outliner from selected control shape</string>
            </property>
            <property name="text">
             <string>Remove Color in Outliner</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="setColorInOutlinerButton">
            <property name="toolTip">
             <string>sets outliner color of selected shape based on color selected above</string>
            </property>
            <property name="statusTip">
             <string>Set color of selected controls name in outliner</string>
            </property>
            <property name="text">
             <string>Set Color in Outliner</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="7" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="C:/Users/julian.beiboer/Desktop/testQTResources/test.qrc"/>
 </resources>
 <connections/>
</ui>
